streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/watcher/FlinkAppHttpWatcher.java

```diff
@@ -679,7 +678,7 @@ private Overview httpOverview(Application application) throws IOException {
         && (FlinkExecutionMode.YARN_APPLICATION == application.getFlinkExecutionMode()
             || FlinkExecutionMode.YARN_PER_JOB == application.getFlinkExecutionMode())) {
       String reqURL;
-      if (StringUtils.isBlank(application.getJobManagerUrl())) {
+      if (StringUtils.isBlank(application.getJobManagerUrl()) || ! application.getJobManagerUrl().contains("http") ) {
```

Comment:
Check if the URL is a valid http url right? if so, We can add new method in common/Utils.scala，then call the method

```
def checkHttpURL(urlString: String) = {
    Try {
      val url = new URL(urlString)
      val connection = url.openConnection.asInstanceOf[HttpURLConnection]
      connection.setRequestMethod("HEAD")
      connection.getResponseCode == HttpURLConnection.HTTP_OK
    }.getOrElse(false)
  }
```